web/css/src/themes/default.css

```diff
@@ -239,14 +240,16 @@
 .top-bar-notification-delete-all {
 	display: block;
 	background-color: transparent;
-	border: 0;
-	color: inherit;
-	width: 100%;
 	text-align: center;
+	border: 0;
 	padding: 10px 15px;
+	width: 100%;
+	background: none;
```

Comment:
Doesn't look like any need to set shorthand here, `background-color` should do?

Reply:
indeed, I'll change that. I have a bad habit of using shorthands unnecessarily that I'm trying to change haha